ports/tensorflow-c-gpu/portfile.cmake

```
@@ -0,0 +1,44 @@
+# # Specifies if the port install should fail immediately given a condition
+vcpkg_fail_port_install(MESSAGE "tensorflow-c-gpu currently only supports Windows platforms" ON_TARGET "UWP" "LINUX" "ANDROID" "FREEBSD" "OSX")
+vcpkg_fail_port_install(MESSAGE "tensorflow-c-gpu currently only supports x64 architecture" ON_ARCH "x86" "arm" "arm64")
+vcpkg_fail_port_install(MESSAGE "tensorflow-c-gpu currently does not support static libraries" ON_LIBRARY_LINKAGE "static")

```

```suggestion
vcpkg_fail_port_install(ON_ARCH "x86" "arm" ON_TARGET "UWP" "LINUX" "ANDROID" "FREEBSD" "OSX")

vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
```
We use `vcpkg_check_linkage()` to judge if the port supports static or dynamic build.

---

@NancyLi1013 thanks for the suggested change. I will commit as you have suggested.